openfecwebapp/config.py

```diff
@@ -15,7 +15,7 @@
 transition_url = env.get_credential('FEC_TRANSITION_URL', 'https://transition.fec.gov')
 classic_url = env.get_credential('FEC_CLASSIC_URL', 'http://classic.fec.gov')
 cms_url = env.get_credential('FEC_CMS_URL', '')
-canonical_base = 'https://beta.fec.gov'
+canonical_base = env.get_credential('CANONICAL_BASE', 'https://beta.fec.gov')
```

Comment:
Oh yeah we'll definitely set it there, but that default value going forward should reflect what it ought to be as well if it can't be found.